### PR TITLE
Add impact to career ladder

### PR DESCRIPTION
**Pull request by hcarreras**
I recommend to "Hide whitespace changes" when reviewing this PR.

### CODE REVIEW

File: career-development/ladder.md [hunk 1]
```diff
@@ -133,23 +169,35 @@ Here are some examples of other career ladders:
...
 
-#### Iterative	
+#### Iterative
 - [ ] Can guide an IPM and team reflection.
-#### Communication	
+
+#### Communication
 - [ ] Aligns team on expectations.
 - [ ] Identifies miscommunications and solves them before they become a problem.
+-  Identifies and proposes strategies around technical problems affecting their team and communicates standards.
```
Comment (jeppester) [inline career-development/ladder.md:178]: Missing `[ ]`

**Comment by jeppester** [comment]
Great additions :+1:

File: career-development/ladder.md [hunk 2]
```diff
@@ -17,114 +17,150 @@ Here are some examples of other career ladders:
 1. Heavily Team Oriented
 1. High Employee Retention
 1. High Quality Code Base
-1. Open Communication 
+1. Open Communication
 1. Pair-Programs
 
-## Dev Values : Simple, Test-Driven, Iterative, Communicative, Respectful, Skilled 
+## Dev Values : Simple, Test-Driven, Iterative, Communicative, Respectful, Skilled
```
Comment (heatherm) [inline career-development/ladder.md:23]: Add `Impact`

File: career-development/ladder.md [hunk 3]
```diff
@@ -17,114 +17,150 @@ Here are some examples of other career ladders:
...
 - [ ] Knows about the three main sections of a Git project(working, staging, repository) as well as the most common commands(commit, pull, push)
 - [ ] Uses a common linter such as rubocop
 - [ ] Can write code in at least one language.
 - [ ] Broad knowledge of main CS + XP concepts.
 
 ### Level 2
-#### Simple 	
+
+#### Impact
+- [ ] Capable of providing on-call support for their area including systems that they are not familiar with.
```
Comment (heatherm) [inline career-development/ladder.md:59]: I would like to reframe the addition of 'on-call' support, as I'm working away from that model and more toward a highly scoped 30 hour week. Perhaps just 'support' vs 'on-call support' there?

File: career-development/ladder.md [hunk 4]
```diff
@@ -17,114 +17,150 @@ Here are some examples of other career ladders:
...
 #### Technical skills
 - [ ] Understands what differentiates their favourite language with others. Knows what the language is useful for and when it is not the best option.
 - [ ] Understand where to place logic in their favourite framework.
 
 ### Level 3
-#### Simple 
-- [ ] -
+
+#### Impact
+- [ ] End-to-end responsibility on projects of increasing complexity.
```
Comment (heatherm) [inline career-development/ladder.md:90]: Let's flesh out end-to-end a bit more. Perhaps dev-ops to engineering or engineering-technical PMing as two ranges to work with?

File: career-development/ladder.md [hunk 5]
```diff
@@ -17,114 +17,150 @@ Here are some examples of other career ladders:
...
+
+#### Simple
+- [ ] Capable of prioritizing tasks. Avoids getting caught up in unimportant details and focuses on delivering value to the client.
```
Comment (heatherm) [inline career-development/ladder.md:93]: To me, this points directly to:

```
bikeshedding

1. Futile investment of time and energy in discussion of marginal technical issues.
2. Procrastination.
```
Could we specify that in particular here?

File: career-development/ladder.md [hunk 6]
```diff
@@ -17,114 +17,150 @@ Here are some examples of other career ladders:
...
 
 ### Level 5
-#### Simple 	
-- [ ] Extracts complex logic into small easy-to-follow classes/components.
+
+#### Impact
+- [ ] It is recognized as a helpful consultant that consistently delivers high-value solutions for real problems.
+
+#### Simple
+- [ ] Creates architecture that enables many potential futures without knowing exactly what the future is.
```
Comment (heatherm) [inline career-development/ladder.md:161]: This one is really nicely stated 🌮

File: career-development/ladder.md [hunk 7]
```diff
@@ -17,114 +17,150 @@ Here are some examples of other career ladders:
...
+- [ ] Understands the broad architecture of the entire system and they are capable of explaining the technical decisions that were taking to tackle the client's problems.
```
Comment (heatherm) [inline career-development/ladder.md:162]:
```suggestion
- [ ] Understands the broad architecture of the entire system and are capable of explaining the technical decisions taken by the team.
```

File: career-development/ladder.md [hunk 8]
```diff
@@ -133,23 +169,35 @@ Here are some examples of other career ladders:
...
 - [ ] Breaks down complex projects into simple systems that can be effectively built and maintained by less experienced developers.
 - [ ] Sought out for technical guidance.
 - [ ] Anticipates technical issues at a product level and works with design to avoid them.
 
 ### Level 6
-#### Simple 	
+
+#### Impact
+- [ ] Has made an obvious positive impact on the entire company's technical trajectory.
+- [ ] Focused on and responsible for their team's productivity.
```
Comment (heatherm) [inline career-development/ladder.md:193]:
```suggestion
- [ ] Focused on and responsible for their team's productivity and growth.
```

File: career-development/ladder.md [hunk 9]
```diff
@@ -133,23 +169,35 @@ Here are some examples of other career ladders:
...
+
+#### Simple
 - [ ] Has a clear understanding of long term maintenance costs and tradeoffs related to simplifying code
-#### Test-driven	
+- [ ] Consistently able to reduce the complexity of projects, services, and processes in order to get more done with less work.
```
Comment (heatherm) [inline career-development/ladder.md:197]: Also a very nice addition!

File: career-development/ladder.md [hunk 10]
```diff
@@ -17,114 +17,150 @@ Here are some examples of other career ladders:
...
 #### Technical skills
 - [ ] Demonstrates ability to succeed in a wide range of complex situations across multiple axes: e.g. scale, uncertainty, interconnectedness.
 - [ ] Researches and proposes new technologies to the team(s).
 
 ### Level 5
-#### Simple 	
-- [ ] Extracts complex logic into small easy-to-follow classes/components.
+
+#### Impact
+- [ ] It is recognized as a helpful consultant that consistently delivers high-value solutions for real problems.
```
Comment (runephilosof-abtion) [inline career-development/ladder.md:158]:
```suggestion
- [ ] Is recognized as a helpful consultant that consistently delivers high-value solutions for real problems.
```

File: career-development/ladder.md (hunk 1 above)
Comment (runephilosof-abtion) [reply career-development/ladder.md:178]:
```suggestion
-  [ ] Identifies and proposes strategies around technical problems affecting their team and communicates standards.
```

File: career-development/ladder.md (hunk 5 above)
Comment (hcarreras) [reply career-development/ladder.md:93]: After some research, I love the term. However, I want to make sure that you can understand the ladder without the need to google. I'll add both, term, and the description.

File: career-development/ladder.md (hunk 4 above)
Comment (hcarreras) [reply career-development/ladder.md:90]: Good point. What about?
Level 3:
- From internal meetings to internal acceptance
- Both backend and frontend implementation
Level 4:
- From client (research) to client acceptance
- Backend and frontend implementation

File: career-development/ladder.md [hunk 11]
```diff
@@ -17,114 +17,151 @@ Here are some examples of other career ladders:
...
 #### Technical skills
 - [ ] Understands what differentiates their favourite language with others. Knows what the language is useful for and when it is not the best option.
 - [ ] Understand where to place logic in their favourite framework.
 
 ### Level 3
-#### Simple 
-- [ ] -
+
+#### Impactful
+- [ ] Take task from internal meetings to internal acceptance.
```
Comment (hcarreras) [inline career-development/ladder.md:90]: I'm struggling with the wording on this one. Suggestions are welcome.

File: career-development/ladder.md [hunk 12]
```diff
@@ -115,13 +115,14 @@ Here are some examples of other career ladders:
...
+- [ ] Can write code in at least two technical areas (frontend, backend, digital infrastructure). Ensures the completion of the tasks in their two areas of expertise.
 - [ ] Can debug issues occuring across multiple levels of the stack.
 
 ### Level 4
 
-#### Impact
-- [ ] Partners with project manager and designers to drive technical decisions based on highest output for all parties needs.
+#### Impactful
+- [ ] Partners with project managers and designers to drive technical decisions based on the highest output for all parties' needs.
+- Take tasks from client research to client acceptance.
```
Comment (runephilosof-abtion) [inline career-development/ladder.md:125]:
```suggestion
- [ ] Take tasks from client research to client acceptance.
```

File: career-development/ladder.md (hunk 11 above)
Comment (runephilosof-abtion) [reply career-development/ladder.md:90]:
```suggestion
- [ ] Drives tasks from internal meetings to internal acceptance.
```

Comment (runephilosof-abtion) [reply career-development/ladder.md:90]:
```suggestion
- [] Picks up tasks from internal meetings and parks them nicely in internal acceptance.
```

**Pull request merged**